.github/workflows/rhub.yaml

```
@@ -0,0 +1,115 @@
+# This workflow is based on R-hub's generic GitHub Actions workflow file.
+# Its canonical location is at
+# https://github.com/r-hub/actions/blob/main/workflows/rhub.yaml
+
+name: R-hub 🌐
+
+on:
+  workflow_call:
+    inputs:
+      config:
+        description: 'A comma separated list of R-hub platforms to use.'
+        type: string
+        # These platforms correspond roughly to CRAN package checks.
+        # More information: https://github.com/insightsengineering/idr-tasks/issues/781
+        # All supported R-hub platforms can be viewed by running: rhub::rhub_platforms()
+        # Container-based platforms (as opposed to VM-based Windows and macOS platforms)
+        # can be also viewed here: https://r-hub.github.io/containers/containers.html
+        default: 'ubuntu-clang,ubuntu-gcc12,gcc13,ubuntu-next,ubuntu-release,macos-arm64,macos,windows'
+      lookup-refs:
+        description: |
+          List of package references to be used for the feature branch.
+          Multiple entries in new lines or separated by commas.
+        required: false
+        default: ""
+        type: string
+
+jobs:
+  setup:
+    runs-on: ubuntu-latest
+    outputs:
+      containers: ${{ steps.rhub-setup.outputs.containers }}
+      platforms: ${{ steps.rhub-setup.outputs.platforms }}
+
+    steps:
+    - uses: r-hub/actions/setup@v1
+      with:
+        config: ${{ inputs.config }}
+      id: rhub-setup
+
+  linux-containers:
+    needs: setup
+    if: ${{ needs.setup.outputs.containers != '[]' }}
+    runs-on: ubuntu-latest
+    name: ${{ matrix.config.label }}
+    strategy:
+      fail-fast: false
+      matrix:
+        config: ${{ fromJson(needs.setup.outputs.containers) }}
+    container:
+      image: ${{ matrix.config.container }}
+
+    steps:
+      - uses: r-hub/actions/checkout@v1
+
+      - uses: r-hub/actions/platform-info@v1
+        with:
+          token: ${{ secrets.RHUB_TOKEN }}
+          job-config: ${{ matrix.config.job-config }}
+
+      - name: Setup R dependencies
```

We are skipping install here so it's more suitable to state the following:
```suggestion
      - name: Prepare DESCRIPTION file
```